Assets/MixedRealityToolkit/_Core/Devices/VoiceInput/WindowsDictationInputDeviceManager.cs

```diff
@@ -59,8 +59,14 @@ public class WindowsDictationInputDeviceManager : BaseDeviceManager, IMixedReali
         /// </summary>
         private AudioClip dictationAudioClip;
 
-        private DictationRecognizer dictationRecognizer;
-        
+        private static DictationRecognizer dictationRecognizer;
```

Comment:
What's the benefit to adding `static` here?

Reply:
because then I couldn't call it in the non-static methods

Reply:
So that I can use it in the other static contexts where it is needed.

Reply:
For posterity (ex: when someone does a search and finds this 6 months from now), can you provide and example of one of the static contexts where you need access?

thx!

Reply:
Sure, two lines below, in the [readonly fields where we setup our async helpers.](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/2921/files#diff-ee543afc5d05dda8c9247ad49c1ab8e7R67)

Reply:
There should never be more than one dictation recognizer or `WindowsDictationInputDeviceManager` running in an application anyhow.

Reply:
that's what I figured. thanks for making it clear for future readers :)